src/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 8
+cache:
+  directories:
+    - node_modules
+script:
+  - npm run build
+  - npm test
```

Comment:
you need also to specify which branches these test are going to run. i would advice you start with `develop` and `master`

Reply:
see this example https://github.com/hisptz/idashboard/blob/2.1/.travis.yml#L7